README.md

```diff
@@ -10,14 +10,13 @@ npm install -g bitcore-node
 bitcore-node start
 ```
 
-Note: For your convenience, we distribute binaries for x86_64 Linux and x86_64 Mac OS X. Upon npm install, the binaries for your platform will be downloaded. For more detailed installation instructions, or if you want to compile the project yourself, then please see the [Build & Install](docs/build.md) documentation to build the project from source.
+Note: For your convenience, we distribute bitcoind binaries for x86_64 Linux and x86_64 Mac OS X. Upon npm install, the binaries for your platform will be downloaded. For more detailed installation instructions, or if you want to compile the project yourself, then please see the Bitcore branch of [Bitcoin Core with additional indexing](https://github.com/bitpay/bitcoin/tree/0.12-bitcore).
 
 ## Prerequisites
 
 - Node.js v0.12 or v4.2
-- ~100GB of disk storage
+- ~150GB of disk storage
 - ~4GB of RAM
```

Comment:
when all is sync'ed and indexed, the final size of just the data directory was 149GB. I would bump this to 200GB.